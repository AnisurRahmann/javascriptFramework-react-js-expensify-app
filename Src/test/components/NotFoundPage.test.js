import React from 'react';
import NotFoundPage from '../../components/NotFoundpage';
import {shallow} from 'enzyme';

test('Not Foun page testing',()=>{
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
})