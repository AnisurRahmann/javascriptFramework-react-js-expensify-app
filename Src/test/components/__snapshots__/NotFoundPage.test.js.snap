// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Foun page testing 1`] = `
<div>
  404 - 
  <Link
    replace={false}
    to="/"
  >
     go Home
  </Link>
</div>
`;
